libs/api-client-bear/src/util.ts

```diff
@@ -35,6 +36,7 @@ export interface IPollingOptions {
     attempts?: number;
     maxAttempts?: number;
     pollStep?: number;
+    blobContentType?: string;
```

Comment:
Can you describe each property ? I know that it  internal type but it will be handy ...

Reply:
Fixed